Jenkinsfile_CNP

```diff
@@ -173,8 +173,8 @@ withPipeline("nodejs", product, component) {
     slackSend(channel: '#finrem-ccd-thread-block', color: 'warning', message: "Master config has been uploaded to AAT. Build details here: <${env.RUN_DISPLAY_URL}|Build ${env.BUILD_DISPLAY_NAME}>.")
   }
 
-  onPR {
```

Comment:
Moving forward, if you wish to retain your pods for testing, add a new label to your FR CCD Defs PR in github called: no_cleanup.  This change means that you no longer need to alter Jenkinsfile_CNP to preserve your PR environment. - baiscally you can revert this to master. Just create Label. - if unsure how to create label on PR. Anyone can show you.. :) just suggestion.